sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileAsyncClient.java

```diff
@@ -216,6 +217,7 @@ public String getFileName() {
     /**
      * Flushes (writes) data previously appended to the file through a call to append.
      * The previously uploaded data must be contiguous.
+     * By default this method will not overwrite existing data.
```

Comment:
You'll want to separate the lines with `<p>` tags since Javadocs are generated as HTML and plain character newlines aren't respected.

Reply:
Done